pavelib/assets.py

```diff
@@ -191,31 +221,59 @@ def compile_sass(options):
     """
     Compile Sass to CSS.
     """
+
+    # Note: import sass only when it is needed and not at the top of the file.
```

Comment:
It is a little ugly, but we do it in other places in the code where we need to be careful about load order. I don't think we can ever move it up, unless we switch from using Paver to install the prereqs on sandboxes, Jenkins etc. This seems like a good argument for using makefiles as we do in all (most?) of our other repos.